### DISCUSSION TITLE
Typing "L" seems to cause freeze

### DISCUSSION THREAD

**Discussion by gwww** [jesseduffield/lazygit#2387] [General]
I don't know if it is just me and user error, but if the first thing I type after opening lazygit is an "L" then I cannot do anything else. q, esc, don't work. Nothing I seem to type works. It happens every time I try it. I resort to killing the process to get out of lazygit.

I'm running the latest version from brew, Silicon Mac latest MacOS, no custom config for lazygit.

**Comment by mark2185** [comment]
Yup, it's a [known](https://github.com/jesseduffield/lazygit/issues/2289) issue.
